util/CamReader.py

```diff
@@ -0,0 +1,109 @@
+import random
+import string
+from os import system
+from time import sleep
+
+import cv2 as cv
+import numpy as np
+from PIL import Image
+from sklearn.cluster import KMeans
+
+
+class CamReader():
+    """Utility class to operate camera hardware and capture"""
+
+    def __init__(self):
+        self.cap = self._open_camera()
+
+    @staticmethod
+    def _open_camera() -> cv.VideoCapture:
+        """Opens the camera"""
+        #  0 -> camera number, if external camera is installed this number needs to be changed
```

Comment:
take `0` as default arg. we can ask user to input camera number from our main.py's args

Reply:
hmm, will read about it